stravalib/tests/functional/test_client_write.py

```diff
@@ -40,7 +40,7 @@ def test_update_activity(self):
         now = datetime.now().replace(microsecond=0)
         a = self.client.create_activity(
             "test_update_activity#create",
-            activity_type=model.Activity.RIDE,
```

Comment:
These constants don't exist anymore. We may need to support these if any user code is highly dependent on them.